src/gui/accountsettings.cpp

```diff
@@ -55,13 +55,34 @@
 #include <QVariant>
 #include <QJsonDocument>
 #include <QToolTip>
-#include <qstringlistmodel.h>
-#include <qpropertyanimation.h>
 
 #include "account.h"
 
 namespace {
 constexpr auto propertyFolderInfo = "folderInfo";
+
+bool showEnableE2eeWithVirtualFilesWarningDialog(QObject *parent)
+{
+    Q_ASSERT(parent);
+    QMessageBox e2eeWithVirtualFilesWarningMsgBox;
+    e2eeWithVirtualFilesWarningMsgBox.setText(parent->tr("End-to-End Encryption with Virtual Files"));
+    e2eeWithVirtualFilesWarningMsgBox.setInformativeText(parent->tr("You seem to have the Virtual Files feature enabled on this folder. At "
+                                                                    " the moment, it is not possible to implicit downloading virtual files that are "
```

Comment:
I think it is "to implicit download" not "to implicit downloading".

Reply:
> I think it is "to implicit download" not "to implicit downloading".

@FlexW @camilasan I'd add one more improvement to Camila's suggestion. Should have **ly** at the end:
implicit**ly**